app/controllers/submissions_controller.rb

```diff
@@ -18,7 +18,9 @@ class SubmissionsController < ApplicationController
     p.frame_src(*PERMITTED_IFRAME_SRC)
   end
 
-  content_security_policy_report_only only: :html_content
+  content_security_policy only: :html_content do |p|
```

Comment:
I just tested this out and I don't think this change is necessary after all, if you fix the script loading order I mentioned in `html_content.html.erb`. But you should test this out yourself to verify and then (assuming you see the same behaviour) revert this change.